src/Binding/JSModules.h

```diff
@@ -64,7 +64,7 @@ class JSModule
 
 private:
     JSContext *m_Cx;
-
+    void * m_dlmodule;
```

Comment:
Our coding convention for members is `m_VariableName` (first letter is upper case). I suggest, `m_DLModule` (for DynamicLoadedModule). Also initilization should be done here too, so `m_DLModule = nullptr;`